components/brave_wallet/browser/keyring_service.cc

```diff
@@ -656,7 +670,10 @@ void KeyringService::CreateWallet(const std::string& password,
                                   CreateWalletCallback callback) {
   auto* keyring = CreateKeyring(mojom::kDefaultKeyringId, password);
   if (keyring) {
-    AddAccountForDefaultKeyring(GetAccountName(1));
+    AddAccountForKeyring(mojom::kDefaultKeyringId, GetAccountName(1));
+  }
+  if (IsFilecoinEnabled()) {
+    CreateKeyring(mojom::kFilecoinKeyringId, password);
```

Comment:
This should be removed since we are doing lazy creation

Reply:
done